### ISSUE TITLE
Lab 2 Question 6

### ISSUE THREAD

**Issue by Schlinkert** [DS4PS/cpp-523-sum-2020#8]
I received a question about question six in lab two.

Without completely giving away the answer, I want to point out that it would be incredibly complicated to solve this problem with the same formula used in Model 4. The reason being, this formula does not account for additional control variables (there is a formula that can do this, but it gets really complicated really quickly).

The important thing to note is that as you add additional control variables (1) the slope of classroom size likely changes, and (2) consequently we need a more complicated formula to calculate the slope when controls are involved.

We will not calculate slopes from a multiple regression model by hand in this class, but just remember that the basic formula cov(x,y)/var(x) is meant to provide intuition about the models, but it will not give you a precise answer any time control variables are included in the model.

**Comment by ecking** [comment]
Ok so for question 6, you’re looking for an explanation on if the covariance formula would work in this case? You’re not actually looking for us to solve for the slope given that there are multiple variables involved, is that right? I went down the path of trying to find a formula 😆 and it was quite hard.

**Comment by Schlinkert** [comment]
Correct. It is important to notice that the cov(x,y)/var(x) formula ONLY works in a bivariate model. But, the intuition behind it, that we can break down a multivariate model to look at the impact of one independent variable on one dependent variable is the key takeaway. This intuition will become even more important as you start building models where you add or omit variables.
